sdk/python/kfp/compiler/compiler.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018-2019 The Kubeflow Authors
```

Comment:
I think we need to break this PR into 2 at least, otherwise the change history chain will be broken.
So maybe in the first PR, move everything v1 to a `v1` folder. Then in a following PR, move everything v2 to the top level folder, that way we'd preserve the history chain, optionally delete `v1` or do the deletion in a third PR.

This also helps keep the current sample test working, you can update the test code to import from `kfp.v1`. It won't be a gigantic PR that changes too many things at a time.

Reply:
`v1` folder`, or `legacy`, or `deprecated`, WDTY?